### PR TITLE
fix python indentation to 4 spaces, set qt default path to '/usr'

### PR DESCRIPTION
**Pull request by MarcAntoine-Arnaud**
using scons with:
`scons --qt-suffix=qt4/`
Build crashed because `qt_dir` is None

### CODE REVIEW

File: SConstruct [hunk 1]
```diff
@@ -58,6 +59,7 @@ AddOption(
     '--qt',
     dest='qt',
     type='string',
+    default='/usr',
```
Comment (cchampet) [inline SConstruct]: Can you extract this line in a single commit? The other changes are not necessary: 4 spaces in python is the [PEP8 recommendation for indentation](https://www.python.org/dev/peps/pep-0008/#indentation).

Comment (MarcAntoine-Arnaud) [inline SConstruct]: agreed to split in 2 commits.
For 4 spaces, it's not necessary but in this file 4 and 8 spaces are used.

Comment (cchampet) [inline SConstruct]: Oups your are right I did not see that you clean the indentation this way. I've also done that at commit 164378e8fccf7336af85b9fe3360b5996b6dc15c

**Comment by MarcAntoine-Arnaud** [comment]
replaced with: https://github.com/mikrosimage/loudness_validator/pull/52

**Pull request closed without merging**